pkg/repositories/transformers/project.go

```diff
@@ -14,6 +14,14 @@ type CreateProjectModelInput struct {
 
 func CreateProjectModel(project *admin.Project) models.Project {
 	stateInt := int32(project.State)
+	if project.Labels == nil {
+		return models.Project{
+			Identifier:  project.Id,
+			Name:        project.Name,
+			Description: project.Description,
+			State:       &stateInt,
+		}
+	}
 	projectBytes, err := proto.Marshal(project)
```

Comment:
Can you explain why this check is needed . Seems even if Labels is not passed the proto.Marshal should generate a byte string which is used as labels on L33

Reply:
When I was running integration tests, I found that we will lose labels if the update project request didn't have labels.
We tried to assign `project.label` a `projectBytes` that didn't contain labels, so gorm will update labels value to nil.